scene/main/viewport.h

```diff
@@ -355,7 +360,8 @@ class Viewport : public Node {
 		Control *last_mouse_focus = nullptr;
 		Control *mouse_click_grabber = nullptr;
 		MouseButton mouse_focus_mask = MouseButton::NONE;
-		Control *key_focus = nullptr;
+		Vector<FocusedControl> focused_controls;
```

Comment:
```suggestion
		LocalVector<FocusedControl> focused_controls;
```
Can be local vector if you are not passing it anywhere, AFAIK.

Reply:
The iterator was added, so I think this can be replaced now.

Reply:
This is outdated, the new version uses a HashMap.
`HashMap<int, Control *> focused_controls;`